File: day006/day006-1.c
#include <stdio.h>

int main() {
	int x, y;
	
	printf("Bir sayı giriniz ");
	scanf("%d", &x);
	
	printf("Bir sayı giriniz ");
	scanf("%d", &y);
	
	printf("%d \n", (x+y));
	printf("%d \n", (x-y));
	printf("%d \n", (x*y));
	printf("%d \n", (x/y));
	
	return 0;
}

File: day006/day006-1.cs
using System;

	class Day006 {
		static void Main() {
			float x, y;
			Console.Write("Bir sayı giriniz: X ");
			x = Convert.ToInt32(Console.ReadLine());
			Console.Write("Başka bir sayı daha giriniz: Y ");
			y = Convert.ToInt32(Console.ReadLine());
			Console.WriteLine(x+y); 
			Console.WriteLine(x-y);
			Console.WriteLine(x*y);
			Console.WriteLine(x/y);   
		}
}
